packages/babel-helpers/src/helpers/regeneratorRuntime.ts

```diff
@@ -347,23 +420,24 @@ export default function _regeneratorRuntime() {
   // result, either by returning a { value, done } result from the
   // delegate iterator, or by modifying context.method and context.arg,
   // setting context.delegate to null, and returning the ContinueSentinel.
-  function maybeInvokeDelegate(delegate, context) {
-    var methodName = context.method;
-    var method = delegate.iterator[methodName];
+  function maybeInvokeDelegate(delegate: Delegate, context: Context) {
+    var methodName = context.method!;
+    var method = delegate[0][methodName];
     if (method === undefined) {
       // A .throw or .return when the delegate iterator has no .throw
       // method, or a missing .next method, always terminate the
       // yield* loop.
       context.delegate = null;
 
       // Note: ["return"] must be used for ES3 parsing compatibility.
-      if (methodName === "throw" && delegate.iterator["return"]) {
+      if (methodName === "throw" && delegate[0]["return"]) {
```

Comment:
Can we keep delegate being an object but mangling its property names? Doing so should be shorter than number index and slightly more readable: i for iterator vs 0 for iterator.

Reply:
After I made the change I realized that this doesn't save any bytes due to `this.delegate = { i: values(iterable), r: resultName, n: nextLoc }`, but that's ok.